src/gobupload/apply/main.py

```diff
@@ -31,21 +31,20 @@ def apply_events(storage: GOBStorageHandler, last_events: set[str], start_after:
     :param stats: update statitics for this action
     :return:
     """
+    CHUNK_SIZE = 10_000
+
     with (
-        ProgressTicker("Apply events", 10_000) as progress,
+        ProgressTicker("Apply events", CHUNK_SIZE) as progress,
         EventApplicator(storage, last_events, stats) as event_applicator,
     ):
-        for chunk in storage.get_events_starting_after(start_after):
+        while chunk := storage.get_events_starting_after(start_after, CHUNK_SIZE):
             with storage.get_session():
                 for event in chunk:
                     progress.tick()
                     event_applicator.load(event)
 
-                try:
-                    event_applicator.flush()
-                except Exception as err:
-                    logger.error(f"Exception during applying events: {repr(err)}")
-                    break  # skips 'else' and executes 'finally' => session rollback + closed
+                event_applicator.flush()
+                start_after = event.eventid
```

Comment:
Loop je nog het risico dat `event` hier niet gedefinieerd is?

Reply:
in dat geval zouden we daar niet moeten komen als chunk een lege lijst is -> while False